comp/core/tagger/component.go

```diff
@@ -18,20 +17,23 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/tagger/collectors"
 	"github.com/DataDog/datadog-agent/pkg/tagger/types"
 	"github.com/DataDog/datadog-agent/pkg/tagset"
+	"github.com/DataDog/datadog-agent/pkg/util/fxutil"
+	"go.uber.org/fx"
 )
 
-// Tagger is an interface for transparent access to both localTagger and
-// remoteTagger.
-type Tagger interface {
-	Init(context.Context) error
+type Component interface {
+	Start(ctx context.Context) error
 	Stop() error
-
 	Tag(entity string, cardinality collectors.TagCardinality) ([]string, error)
 	AccumulateTagsFor(entity string, cardinality collectors.TagCardinality, tb tagset.TagsAccumulator) error
 	Standard(entity string) ([]string, error)
 	List(cardinality collectors.TagCardinality) tagger_api.TaggerListResponse
 	GetEntity(entityID string) (*types.Entity, error)
-
 	Subscribe(cardinality collectors.TagCardinality) chan []types.EntityEvent
 	Unsubscribe(ch chan []types.EntityEvent)
 }
+
+// Module defines the fx options for this component.
+var Module = fxutil.Component(
```

Comment:
Module is now a function

Reply:
Yeah. I will pull and merge from latest branch.